integreat_cms/cms/migrations/0084_alter_region_slug.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.2.22 on 2023-11-17 11:01
+
+from django.db import migrations, models
+import integreat_cms.cms.models.regions.region
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0083_add_page_based_offer_fields'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
Question @timobrembeck. When I add a validator for the PageTranslation (via the `@modify_fields` wrapper) no automatic migration is generated. Should we then in general prefer using the form `clean_xyz()`-approach to validate such properties?

Reply:
Hmm, probably yes. Out of the top of my head, I don't have an idea how to overwrite the validators of an existing field if `@modify_fields` doesn't work...

Reply:
It does work. But the migrations don't work out of the box. But never mind. I'll just use the clean approach as it is more intuitive and probably less error prone 👍🏼